### PR TITLE
Regenerate baseline because last one break.

### PR DESCRIPTION
**Pull request by sima-zhu**
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=708674&view=results

### CODE REVIEW

File: eng/pipelines/aggregate-reports.yml [hunk 1]
```diff
@@ -104,13 +104,11 @@ jobs:
...
-        # GdnBreakBaselineFiles: $(Build.SourcesDirectory)\eng\java.gdnbaselines
-        # GdnBreakBaselines: baseline  
+        GdnBreakBaselineFiles: $(Build.SourcesDirectory)\eng\java.gdnbaselines
+        GdnBreakBaselines: baseline  
         # Used for generating baseline file.
-        GdnBreakOutputBaselineFile: java
-        GdnBreakOutputBaseline: baseline
-      condition: succeededOrFailed()
-      continueOnError: true
+        # GdnBreakOutputBaselineFile: java
```
Comment (weshaggard) [inline eng/pipelines/aggregate-reports.yml:110]: What are these here commented out?

Comment (sima-zhu) [reply eng/pipelines/aggregate-reports.yml:110]: The commented codes are used for generating the baseline file. Once we have the file, we only need to run the above ones

File: eng/pipelines/aggregate-reports.yml [hunk 2]
```diff
@@ -98,16 +98,21 @@ jobs:
...
-        GdnBreakBaselines: baseline      
+        GdnBreakBaselineFiles: $(Build.SourcesDirectory)\eng\java.gdnbaselines
+        GdnBreakBaselines: baseline  
+        # Used for generating baseline file.
+        # GdnBreakOutputBaselineFile: java
+        # GdnBreakOutputBaseline: baseline
+      condition: succeededOrFailed()
+    - task: securedevelopmentteam.vss-secure-development-tools.build-task-publishsecurityanalysislogs.PublishSecurityAnalysisLogs@3
+      displayName: 'Publish Security Analysis Logs'
+      continueOnError: true
```
Comment (weshaggard) [inline eng/pipelines/aggregate-reports.yml:115]: Do you want this to continue on error? I thought you baselined things so that shouldn't be needed.

Comment (sima-zhu) [reply eng/pipelines/aggregate-reports.yml:115]: Good call. Will remove

**Review by weshaggard — APPROVED** [review]

**Pull request merged**